boilerplates/react-union-boilerplate-basic/.babelrc

```diff
@@ -1,5 +1,5 @@
 {
   "presets": [
-    "babel-preset-react-union"
```

Comment:
please leave `babel-preset-react-union`. It is bad practice to use a shorter name.

Reply:
Will do